I/ittapi/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ittapi"
+version = v"3.23.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/intel/ittapi.git", "96a01335dbc560b9224f52f0cc25c28ca0f1b564")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ittapi/                                                                                                                                                  
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DITT_API_IPT_SUPPORT=1                                      
+make                                                                                                                                                                          
+ar -x bin/libittnotify.a                                                                                                                                                      
+cc -shared ittnotify_static*.o ittptmark*.o -o libittnotify.so                                                                                                                
+mkdir -p $prefix/lib                                                                                                                                                          
+cp libittnotify.so $prefix/lib/                                                                                                                                               
```

Comment:
To fix macOS, perhaps this will work?
```suggestion
cc -shared ittnotify_static*.o ittptmark*.o -o libittnotify.${dlext}                                                                                                                
mkdir -p $prefix/lib                                                                                                                                                          
cp libittnotify.${dlext} $prefix/lib/                                                                                                                                               
```

Reply:
I prefer using install -Dvm 0755 for moving the file to the prefix, which enforces the rights and saves one command